api/main.py

```diff
@@ -101,6 +102,16 @@ async def get_node(node_id: str):
     return await db.find_by_id(Node, node_id)
 
 
+@app.get('/nodes')
+async def nodes():
+    return {'nodes': await db.find_all(Node)}
+
+
+@app.get('/node', response_model=List[Node])
```

Comment:
In fact I guess this should be `/nodes` since it's returning a number of Node objects rather than just one.